lib/pages/chat_list/chat_list.dart

```diff
@@ -80,13 +73,8 @@ class ChatListController extends State<ChatList>
     with
         TickerProviderStateMixin,
         RouteAware,
-        ComparablePresentationContactMixin {
-  StreamSubscription? _intentDataStreamSubscription;
-
-  StreamSubscription? _intentFileStreamSubscription;
-
-  StreamSubscription? _intentUriStreamSubscription;
-
+        ComparablePresentationContactMixin,
+        ReceiveSharingIntentMixin {
```

Comment:
Why this screen still need this `mixin`?

Reply:
because it has an `initIntent` method, it needs to be called so that the application can know whether the Twake application is clicked in the share menu.